### ISSUE TITLE
Constraint unique on categories

### ISSUE THREAD

**Issue by LePetitTim** [GeotrekCE/Geotrek-admin#3403]
Some categories, types doesn't have constraint unique :

- TouristiccontentCategory
- TouristicEventType
- TouristicContenType1 / 2 (unique together with category)

We should check any categories and type, verify if it needs a unique constraint (or unique together).

For each of this new unique constraint add a migration before the automatic migration which:
-  Merge  all fields possible to be merged from all categories with same label* (B-Z) to the first (A) 
-  Drop all categories (B-Z)

* This is not always the field label (#3006)

**Comment by camillemonchicourt** [comment]
Yes, for me these are mistakes when Touristic content modules has been created, and it would be important to add these constraints.
